commands/about.mjs

```diff
@@ -1,18 +1,28 @@
 // About command to show info about the bot
 
-import { SlashCommandBuilder } from '@discordjs/builders';
 import {
-    MessageEmbed,
+    EmbedBuilder,
+    SlashCommandBuilder
 } from 'discord.js';
 import got from 'got';
 
+import { changeLanguage, t } from '../modules/translations.mjs';
+
 const defaultFunction = {
     data: new SlashCommandBuilder()
         .setName('about')
-        .setDescription('Tells you a bit about the bot'),
+        .setDescription('Tells you a bit about the bot')
+        .setNameLocalizations({
+            'es-ES': 'sobre',
+            ru: 'о',
+        })
+        .setDescriptionLocalizations({
+            'es-ES': 'Te cuenta un poco sobre el bot',
+            ru: 'Рассказывает немного о боте',
```

Comment:
```suggestion
            'ru': 'Рассказывает немного о боте',
```

Should this also be wrapped in quotes?

Reply:
It doesn't really matter since it doesn't have a dash in the field name